SPEC.md

```diff
@@ -626,12 +633,19 @@ Map[String, Int] = {"a": 1, "b": 2}
 
 :pig2: [Cromwell supported](https://github.com/broadinstitute/cromwell#wdl-support) :white_check_mark:
 
-Pair values can be specified using another Python-like syntax, as follows:
+Pair values can be specified inside of a WDL using another Python-like syntax, as follows:
 
 ```
 Pair[Int, String] twenty_threes = (23, "twenty-three")
 ```
 
+Pair values can also be specified within the [workflow inputs JSON](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#specifying-workflow-inputs-in-json) with a `Left` and `Right` value specified using JSON style syntax. For example, given a workflow `wf_hello` and workflow-level variable `twenty_threes`, it could be declared in the workflow inputs JSON as follows:
+```
+{
+  "wf_hello.twenty_threes": { "Left": 23, "Right": "twenty-three" }
```

Comment:
Json to Pair coercion isn't sensitive towards capitalization (there's a test case in WDL4S for it) but I didn't realize the outputs were all lowercase, that creates a slight inconsistency. We could make it required lowercase for the future but currently Pair creation accepts "Left" or "left"